src/org/zaproxy/zap/extension/spider/ExtensionSpider.java

```diff
@@ -119,6 +123,17 @@ private void initialize() {
 		this.customParseFilters = new LinkedList<>();
 		this.scanController = new SpiderScanController(this);
 	}
+	
+	/* */
```

Comment:
Some actual javadoc could be included here (preferably). Or, it could be removed.